transports/quic/src/transport.rs

```diff
@@ -888,4 +906,25 @@ mod test {
             .unwrap();
         assert!(!transport.dialer.contains_key(&SocketFamily::Ipv4));
     }
+
+    #[cfg(feature = "tokio")]
+    #[tokio::test]
+    async fn test_listens_ipv4_ipv6_separately() {
```

Comment:
Can you confirm that this test fails without the fix?

Reply:
yeah I confirmed before submitting, it fails without the fix.